zebra-chain/src/parameters/network.rs

```diff
@@ -274,6 +276,54 @@ impl Network {
             .activation_height(self)
             .expect("Sapling activation height needs to be set")
     }
+
+    /// Returns the expected total value of the sum of all NU6.1 one-time lockbox disbursement output values for this network at
+    /// the provided height.
+    pub fn lockbox_disbursement_total_amount(&self, height: Height) -> Amount<NonNegative> {
+        if Some(height) != NetworkUpgrade::Nu6_1.activation_height(self) {
+            return Amount::zero();
+        };
+
+        match self {
+            Self::Mainnet => subsidy::EXPECTED_NU6_1_LOCKBOX_DISBURSEMENT_TOTAL_MAINNET,
+            Self::Testnet(params) if params.is_default_testnet() => {
+                subsidy::EXPECTED_NU6_1_LOCKBOX_DISBURSEMENT_TOTAL_TESTNET
+            }
+            Self::Testnet(_params) => 0,
```

Comment:
This should be configurable so that this can be tested in regtest/testnet.

Reply:
Done in https://github.com/ZcashFoundation/zebra/pull/9603/commits/271a20de6046196235ea81d124dac12948d66a68.